src/components/structures/login/ForgotPassword.js

```diff
@@ -150,7 +148,8 @@ module.exports = React.createClass({
         var ErrorDialog = sdk.getComponent("dialogs.ErrorDialog");
         Modal.createDialog(ErrorDialog, {
             title: title,
-            description: body
+            description: body,
+            button: _t("OK"),
```

Comment:
i think the "OK" i18n should be done on BaseDialog itself, to avoid having to repeat it everywhere?